scijava/scijava-ops/src/main/java/org/scijava/param/JavadocParameterData.java

```diff
@@ -175,6 +187,19 @@ private boolean hasVanillaJavadoc(MethodJavadoc doc, long numParams, long numRet
 		return sufficientParams && sufficientReturn;
 	}
 
+	/**
+	 * Determines whether {@code doc} has enough

```

Comment:
It still worries me that we can only compare the _number_ here, because we don't have access to the parameter names. Would be much better to do keyword matching. Then if the number differs, you have what you have, and the other descriptions are just missing.

Do we have the parameter names at annotation processing time? If so, we could record them along with the javadoc. Would require changes to therapi, sure, but worth discussing after this work is merged. File an issue for it?

Reply:
> Do we have the parameter names at annotation processing time?

[This StackOverflow post](https://stackoverflow.com/questions/37319564/inspecting-parameter-names-from-annotation-processor) suggests so.

Reply:
I added this objective to scijava/scijava#65